src/components/BottomSheet/BottomSheet.js

```diff
@@ -13,34 +14,31 @@ const BottomSheet = React.forwardRef((props, ref) => {
       onOpen={() => (props.onOpen ? props.onOpen() : {})}
       closeOnDragDown={true}
       closeOnPressMask={props.closeOnPressMask}
-      height={props.height ? props.height : 260}
+      height={props.height ? props.height : 355}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default height of the BottomSheet has been increased from 260 to 355. Make sure this change does not affect the layout or usability of the application on smaller devices or in different orientations (landscape/portrait).
```diff
-      height={props.height ? props.height : 260}
+      height={props.height ? props.height : 355}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->